persistence/3.0/_index.md

```diff
@@ -1,24 +1,24 @@
 ---
 title: "Jakarta Persistence 3.0 (under development)"
```

Comment:
Let's change "(under development)" to "(milestone)" in the header to clarify what's being delivered.  This is consistent with the PRs that have come before you.  Thanks!

Reply:
done